x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas/schemas.ts

```diff
@@ -105,3 +105,7 @@ export const updated_by = Joi.string();
 export const version = Joi.number()
   .integer()
   .min(1);
+export const actions = Joi.array().default([]);
+export const throttle = Joi.string()
+  .valid('no_actions', 'rule', '1h', '1d', '7d')
+  .default('no_actions');
```

Comment:
As discussed above, this should be something like:

```ts
export const throttle = Joi.string()
  .allow(null)
  .default(null);
```

and then update your unit tests in the validators to ensure this can be set to null as suggested or double check in the in joi docs if this is not correct.